packages/venia-ui/lib/components/Gallery/item.js

```diff
@@ -68,9 +68,11 @@ const GalleryItem = props => {
         </div>
     );
 
-    const ratingAverage = rating_summary ? (
-        <Rating rating={rating_summary} />
-    ) : null;
+    // Hide the Rating component until it is updated with the new look and feel (PWA-2512).
+    const ratingAverage = null;
```

Comment:
Two errors from `yarn run lint` : 
```
  17:8   error  'Rating' is defined but never used                   no-unused-vars
  48:54  error  'rating_summary' is assigned a value but never used  no-unused-vars
```
Perhaps remove lines or add a linter disable line?

Reply:
Since we're going to re-enable this soon I just added linter disable lines.

I also added "remove the linter disable lines" to the scope of PWA-2512.